test/test.js

```diff
@@ -35,33 +41,41 @@ rimraf(tmp, function() {
 })
 
 // Test begin
-function startTests(filenames) {
-  test('basics', function(t) {
+function startTests (filenames) {
+  test('basics', function (t) {
     t.assert(called, 'callback is called')
     t.end()
   })
 
-  fs.readdir(outputPath, function(err, files) {
-    if (err) return console.log(err)
-    test('has the right files', function(t) {
-      t.equal( files.length, 5, 'created 5 files?')
-      filenames.forEach(function(name) {
-        fs.readFile(path.resolve(outputPath, name + '.html'), function(err, data) {
-          t.error( err, name + '.html exists?')
-          t.ok( data.toString().match('<strong>' + name + '</strong>'), name + '.html contains html including the name?')
+  fs.readdir(outputPath, function (err, files) {
+    if (err) {
+      return console.log(err)
+    }
+
+    test('has the right files', function (t) {
+      t.equal(files.length, 6, 'created 6 files?')
+      filenames.forEach(function (name) {
+        fs.readFile(path.resolve(outputPath, name + '.html'), function (err, data) {
+          t.error(err, name + '.html exists?')
+          t.ok(data.toString().match('<strong>' + name + '</strong>'), name + '.html contains html including the name?')
         })
       })
-      t.ok( files.indexOf('index.html') >= 0, 'index.html exists?')
+
+      t.ok(files.indexOf('main.css'), 'main.css exists?')
+      t.ok(files.indexOf('index.html') >= 0, 'index.html exists?')
       t.end()
     })
   })
 
-  test('flags work', function(t) {
-    fs.readFile(path.resolve(outputPath, 'index.html'), function(err, data) {
-      if (err) return console.log(err)
+  test('flags work', function (t) {
+    fs.readFile(path.resolve(outputPath, 'index.html'), function (err, data) {
+      if (err) {
+        return console.log(err)
+      }
```

Comment:
I'm not a fan of these changes expanding `if (err) return console.log(err)` to multiple lines, I think they create a lot of visual noise from the functional code. Could you undo them please?